sharness/t0060-migration-stress.sh

```diff
@@ -4,6 +4,15 @@ test_description="Test migration 2 to 3 with lots of objects"
 
 . lib/test-lib.sh
 
+# setup vars for tests
+DEPTH=6
+PINTOTAL=2000
+if [ ! -z "$CI" ]; then
```

Comment:
Travis can run longer than 10m, just needs to receive output from the test,
the check is based on "10m since last output received"
On Sun, Mar 20, 2016 at 13:42 Christian Couder <notifications@github.com>
wrote:

> In sharness/t0060-migration-stress.sh
> <https://github.com/ipfs/fs-repo-migrations/pull/29#discussion_r56768373>:
>
> > @@ -4,6 +4,15 @@ test_description="Test migration 2 to 3 with lots of objects"
> >
> >  . lib/test-lib.sh
> >
> > +# setup vars for tests
> > +DEPTH=6
> > +PINTOTAL=2000
> > +if [ ! -z "$CI" ]; then
>
> Maybe if test -n "$CI" ...
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ipfs/fs-repo-migrations/pull/29/files/d97eda26a4087f4377b66666972f3188f03fb328..8f38ad1012191850b34fb54452a95d996d668784#r56768373>
>